sympy/utilities/lambdify.py

```diff
@@ -102,10 +102,10 @@
 NUMEXPR_TRANSLATIONS = {}
 
 # Available modules:
-MODULES = {
+_MODULES = {
     "math": (MATH, MATH_DEFAULT, MATH_TRANSLATIONS, ("from math import *",)),
     "mpmath": (MPMATH, MPMATH_DEFAULT, MPMATH_TRANSLATIONS, ("from mpmath import *",)),
-    "numpy": (NUMPY, NUMPY_DEFAULT, NUMPY_TRANSLATIONS, ("import_module('numpy')",)),
+    "numpy": (NUMPY, {}, {}, ("import numpy",)),
```

Comment:
People might rely on the `NUMPY_TRANSLATIONS` in the module (at least I do in a project). I never thought about that it might be needed by some lambdified code thats not tested. It was good that tests passed without these, but I think you're right. I will reintroduce them.